pkg/kubehound/core/core_grpc_client.go

```diff
@@ -0,0 +1,40 @@
+package core
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DataDog/KubeHound/pkg/config"
+	pb "github.com/DataDog/KubeHound/pkg/ingestor/api/grpc/pb"
+	"github.com/DataDog/KubeHound/pkg/telemetry/log"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func CoreClientGRPCIngest(ingestorConfig config.IngestorConfig, clusteName string, runID string) error {
+	var dialOpt grpc.DialOption
+	if ingestorConfig.API.Insecure {
+		dialOpt = grpc.WithTransportCredentials(insecure.NewCredentials())
+	} else {
+		return fmt.Errorf("insecure connection is not allowed")
+	}
```

Comment:
Does this mean we _can't_ use a secure connection? Will this be changed in the future?

Reply:
yes it is not supported for now, as we need to also push the 101 on how to deploy a certificate. But anyway, there is nothing sensitive transiting (cluster name and run_id). Will implement it later.

Reply:
But then shouldn't the error message be "**secure** connection is not supported" ?
because if I understood correctly:
- if `ingestorConfig.API.Insecure` is `true`, we are in insecure mode
- if it is false, we are **not** in insecure mode, so I would say it would be **secure** mode no?
or did I misundestood smthg?